docs/marketplace/user/merchant-portal-user-guides/202108.0/offers/managing-product-offers.md

```diff
@@ -17,51 +17,37 @@ Review the reference information in this article before you start, or just look
 To create a product offer:
 
 1. Click **Add Offer**.
-2. From the list of concrete products, select the necessary product you want to create an offer for, hover over the three dots, or just click the line and then click **Create Offer**.
+2. From the list of concrete products, select the  product you want to create an offer for.
 3. In the drawer, fill the necessary fields and click **Create**.
```

Comment:
Consider adding information from this PR: https://github.com/spryker/spryker.github.io/pull/211/files Specifically, the info from lines 20-30 (no need to include the detailed info about the fields that you have in the reference information section of course). "Fill the necessary fields" is not a very helpful step, and there is no information about such fields as for example "Always in stock" or "Offer status". Consider adding them in the reference information or here